copter/source/docs/systemid-mode-operation.rst

```diff
@@ -95,3 +95,101 @@ main loop frequency divided by a sub-sample factor.
 | 0            |    0         | 8             |
 +--------------+--------------+---------------+
 
+Identification of a Multicopter
+===============================
+Description of the Approach
+---------------------------
+
+The Parameter :ref:`SID_AXIS<SID_AXIS>` of the system identification mode feeds in the frequency-sweep at different spots inside Ardupilot's controlers.
+The locations of these injection points are shown in the following :ref:`figure<fig-ctrl-sys-ardupilot>` as small chirp symbols.
+At each injection point, the sweep can be added to either the roll, pitch or yaw axis. 
+
+.. image:: ../images/ControlSystem.png
+:name: fig-ctrl-sys-ardupilot
+
+For the purpose of obtaining a dynamic model of only the copter itself (aka plant model), it is first necessary to calculate a transfer function(s) between the control inputs of the copter and its measurable dynamic response.
+In this case, the angular rate controller outputs (:math:`p_{Thrust}`, :math:`q_{Thrust}` and :math:`r_{Thrust}`) represent the control inputs of the copter whereas the gyroscope's measured angular rates represent the system response.
+Since a common application of the created system model consists in optimizing the control parameters, the filtered gyroscope signals are used instead of the raw gyroscope data, as these are the direct inputs to the angular rate controller.
+Thereby, the separate modelling of the filters becomes redundant, which makes it possible to directly connect a model of the angular rate controller to the plant model without any further effort.
+
+.. image:: ../images/RateControl.png
+:name: fig-rate-ctrl
+
+The generation of flight data for the system identification is conducted during hovering flight.
+Therefore, the three axes (namely roll, pitch and yaw) are considered as decoupled.
+According to the :ref:`figure<fig-rate-ctrl>` shown above, which illustrates the general structure of the rate control loop independent of any specific control axis, it is possible to construct a transfer function for each axis:
+
+.. math::
+
+G_{Roll}(s) &= \frac{Y(s)_{Roll}}{U(s)_{Roll}} = p_{Meas}/p_{Thrust} \\
+G_{Pitch}(s) &= \frac{Y(s)_{Pitch}}{U(s)_{Pitch}} = q_{Meas}/q_{Thrust} \\
+G_{Yaw}(s) &= \frac{Y(s)_{Yaw}}{U(s)_{Yaw}} = r_{Meas}/r_{Thrust} \\
+
+As the figure shows, the transfer function of each axis models the system consisting of the motor mixer, the motors, the airframe of the copter and the gyroscopes together with the filters.
+Initially, the frequency-sweeps of :ref:`SID_AXIS<SID_AXIS>` 7-9 are used to inject the test signal at the input of the angular rate controller.
```

Comment:
To get around the problem you experienced in your analysis, I would recommend using inputs 1-3 or I suppose you could use 3-6 as well.  These are making inputs into the attitude controller and so it will recognize the input and not try to attenuate it.  Then you can still perform the open loop analysis using the mixer inputs.